README.md

```diff
@@ -1,2 +1,10 @@
 # instagram-clone-php
-An Instagram clone coded in PHP 7 with MySQLi support
+An Instagram clone coded in PHP 7 with MySQLi support.
+
+Prerequisites:
+1. A database and a webserver with PHP 7 installed
+
+Instructions:
```

Comment:
Instructions too vague and misleading. Need to be more specific.

Reply:
Okay, thanks for the feedback.